javascript/javascript1/week2/index.js

```diff
@@ -0,0 +1,93 @@
+//Fullname 
+function getFullname(firstname, surname, useFormalName, isMale){
+    let fullname = "";
+    if(!firstname){
+        console.log("please write your firstname");
+        return;
+    }
+    if(!surname){
+        console.log("please write your surname");
+        return;
+    }
+    if (useFormalName) {
+        if (isMale){
+          fullname="Lord ";
+        }
+        else{
+            fullname="Miss ";
+        }
+    }
+    fullname = fullname + firstname + " " + surname;
+    return fullname;
+}
+
+const fullname1= getFullname("Besart","Elezi", true,true);
+console.log(fullname1);
+const fullname2= getFullname("Benjamin","");
+console.log(fullname2);
+
+
+
+//Event application
+function getEventWeekday(daysAfter){
+    const weekDays = ["Monday","Tuesday","Wednesday","Thursday","Friday","Saturday","Sunday"];
+    const today = new Date().getDay();
+    const eventToBeHeld = today + daysAfter;
+    const postionInArray = (eventToBeHeld % 7) - 1;
+    return weekDays[postionInArray];
+}
+console.log(getEventWeekday(9));
+
+
+
+//Weather wear
+function whatToWear(temperature){
+    let clothes = null;
+    if(temperature <= 0){
+        clothes = "Boots,gloves,and scarf";
+    }
+    else if(temperature > 0 && temperature <= 17){
+        clothes = "Jacket and pants";
+    }
+    else{
+        clothes = "T-shirt";
+    }
+    return clothes;
+}
+const clothesToWear = whatToWear(177);
+console.log(clothesToWear);
+
+
+
+//Student manager
+const class07Students = ["s1", "s2", "s3", "s4", "s5", "s6"];
+
+function addStudentToClass(studentName) {
+    const classSize = class07Students.length;
```

Comment:
Nice variable naming! 
You already made a function to calculate this number. What do you think on using it here?